package.json

```diff
@@ -108,7 +108,7 @@
                 },
                 "aws.samcli.debug.attach.timeout.millis": {
                     "type": "number",
-                    "default": 30000,
+                    "default": 90000,
```

Comment:
There is also `SAM_LOCAL_PORT_CHECK_RETRY_TIMEOUT_MILLIS_DEFAULT`. Always a good idea to check the call site (`createInvokeTimer`) for where its inputs are coming from.